src/plugin/ipc/ipc.cpp

```diff
@@ -215,10 +215,10 @@ EXTERNC void
 dmtcp_initialize_plugin()
 {
   dmtcp_register_plugin(sshPlugin);
+  dmtcp_register_plugin(eventPlugin);
```

Comment:
Could you please add a comment explaining why it needs to be called here? Probably, just copy paste the description from issue #405.